src/shared/containers/DashboardListContainer.tsx

```diff
@@ -20,6 +24,26 @@ const DashboardList: React.FunctionComponent<DashboardListProps> = ({
   orgLabel,
   projectLabel,
 }) => {
+  const basePath = useSelector((state: RootState) => state.config.basePath);
+  const history = useHistory();
+  const makeResourceUri = (
+    orgLabel: string,
+    projectLabel: string,
+    resourceId: string
+  ) => {
+    return `${basePath}/${orgLabel}/${projectLabel}/resources/${encodeURIComponent(
```

Comment:
you need to remove the `basePath` here. React-router deals with this automatically

```suggestion
    return `/${orgLabel}/${projectLabel}/resources/${encodeURIComponent(
```